### DISCUSSION TITLE
BirdNET web analyzer at https://birdnet.cornell.edu/api/ is producing odd results

### DISCUSSION THREAD

**Discussion by tphakala** [birdnet-team/BirdNET-Analyzer#442] [General]
I have been using BirdNET web analyzer at https://birdnet.cornell.edu/api/ to verify some of BirdNET-Go captures and I have been wondering why web analyzer produces so different results. Today I installed BirdNET Analyzer Win GUI 1.3.1 version and it produces same results as BirdNET-Go.

Here are example results for same file analyzed in Web version and v1.3.1 Windows GUI version

![Snag_2a2b49](https://github.com/user-attachments/assets/52ef0b02-9a1d-4a4a-a05a-42f7125b04ca)

![Snag_2a24a2](https://github.com/user-attachments/assets/25a76510-3a5f-4b29-af85-5cfdc09ad49c)

Web version results are so way off that I would recommend taking it offline until it is fixed

**Comment by Josef-Haupt** [comment]
The web version uses a long outdated version of BirdNET, as far as I know. But people still seem to use it a lot.
@kahst Which BirdNET version is used at https://birdnet.cornell.edu/api/ ?

**Comment by tphakala** [comment]
People are using it because it's easily accessible (no installation required) and they also believe its results accurately represent the BirdNET model's capabilities

**Comment by kahst** [comment]
Ok, I added a few hints on the upload page that specify the model version as V1.0 (the one from the original BirdNET paper). We'll update the demo page and replace it with a Gradio interface just like the Single File analysis tab.
